teamcity-rest-client-api/src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -548,6 +549,19 @@ interface Change {
     val date: Date
 }
 
+interface ChangeFiles {
+    val count: Int
+    val files: List<ChangeFile>
+}
+
+interface ChangeFile {
+    val beforeRevision: String?
+    val afterRevision: String?
+    val changeType: String
+    val file: String?
```

Comment:
What are these properties are about? Do they store path to the file? If so it makes sense to rename them accordingly. And what are the differences between them?